### PR TITLE
Feature/add more attributes to participant

### PR DESCRIPTION
**Pull request by caseyhelbling**
Adds more attributes to a participant that the new mobile apps will need to consume via the sessions endpoint.

### CODE REVIEW

File: src/app/models/sessions_json_builder.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class SessionsJsonBuilder
+
+  def to_hash(session)
```
Comment (jcoyne) [inline src/app/models/sessions_json_builder.rb]: Not a fan of jbuilder?

Comment (caseyhelbling) [inline src/app/models/sessions_json_builder.rb]: Never used it but looks like it could be awesome.

Comment (caseyhelbling) [inline src/app/models/sessions_json_builder.rb]: gonna stand up a cedar14 instance of prod then come back to plug in jbuilder

**Comment by jcoyne** [comment]
I don't think we need to hold this up for jbuilder. If you want to convert this to jbuilder you can do a new PR.

**Pull request merged**